pkg/trace/stats/concentrator_test.go

```diff
@@ -406,10 +409,10 @@ func TestConcentratorStatsCounts(t *testing.T) {
 		testSpan(now, 11, 0, 333, 3, "A1", "resource3", 0, map[string]string{"span.kind": "client"}),
 		testSpan(now, 12, 0, 444, 3, "A1", "resource3", 0, map[string]string{"span.kind": "server"}),
 		// 2 buckets old, part of the first flush
-		testSpan(now, 1, 0, 24, 2, "A1", "resource1", 0, nil),
+		testSpan(now, 1, 100, 24, 2, "A1", "resource1", 0, nil),
 		testSpan(now, 2, 0, 12, 2, "A1", "resource1", 2, nil),
-		testSpan(now, 3, 0, 40, 2, "A2", "resource2", 2, nil),
-		testSpan(now, 4, 0, 300000000000, 2, "A2", "resource2", 2, nil), // 5 minutes trace
+		testSpan(now, 3, 1, 40, 2, "A2", "resource2", 2, nil),
+		testSpan(now, 4, 1, 300000000000, 2, "A2", "resource2", 2, nil), // 5 minutes trace
```

Comment:
Can we avoid making changes to the existing cases? The choice of parent ID here was likely intentional.

Reply:
I understand having to update the other tests where `parent_id` is 0 so we now expect the flag to be set to true.

Again though, let's see if we can avoid making changes to the parent IDs for the input test data.